shuup/front/apps/carousel/locale/en/LC_MESSAGES/django.po

```diff
@@ -203,6 +203,20 @@ msgid ""
 "banner boxes."
 msgstr ""
 
+msgid "Adding the template to your site"
+msgstr ""
+
+msgid ""
+"To add the carousel to your site, you must edit the page using the template "
+"editor. Go to your site and click on 'Edit page' on the page you want to add "
+"the carousel banner. Then add a new plugin row and from the plugin dropdown "
```

Comment:
What happened to step to choose placeholder and open the editor.

Reply:
Add new plugin to placeholder row or cell by selecting the row/cell and then selecting the plugin from dropdown.